Assets/_Scripts/UIScripts/SampleUI.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class SampleUI : MonoBehaviour {
```

Comment:
we don't need three different scripts that do the same thing, these can be unified as an "effect text" script